cirrus/server/cirrus/sdk.py

```diff
@@ -4,14 +4,12 @@
 
 from cirrus_sdk import CirrusClient, NimbusError  # type: ignore
 
-from .settings import context
-
 logger = logging.getLogger(__name__)
 
 
 class SDK:
-    def __init__(self):
-        self.client = CirrusClient(context)
+    def __init__(self, context: str):
+        self.client = CirrusClient(context)  # does not match struct- throw an exception
```

Comment:
What does this comment mean?

Reply:
so CirrusClient is actually coming from nimbus_sdk, if the context we are providing doesn't match what it is expecting, it will throw an error